### DISCUSSION TITLE
자바스크립트 실행 컨텍스트 딥다이브 | 정현수 기술 블로그

### DISCUSSION THREAD

**Discussion by giscus** [junghyeonsu/junghyeonsu.dev.comment#22] [Announcements]
# 자바스크립트 실행 컨텍스트 딥다이브 | 정현수 기술 블로그

딥다이브 실행 컨텍스트 파트를 읽고

https://junghyeonsu-dev.vercel.app/posts/Execution-context

**Comment by KoGaYoung** [comment]
아는 주제임에도 다시 정독할만큼 글 퀄리티가 좋습니다. 좋은 글 써주셔서 감사해요

**Reply by junghyeonsu** [reply]
읽어주셔서 감사합니다
